### PR TITLE
ROX-23029: Add advanced filters component to Node CVE overview page

### PR DESCRIPTION
**Pull request by dvail**
## Description

Replaces the placeholder filter bar on the Node CVEs list page with the Advanced Filters bar.

## Checklist
- [ ] Investigated and inspected CI test results
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

## Testing Performed

image
image
image
image
image

### CODE REVIEW

File: ui/apps/platform/src/Components/CompoundSearchFilter/types.ts [hunk 1]
```diff
@@ -323,10 +323,10 @@ export const nodeCVESearchFilterConfig = {
     displayName: 'Node CVE',
     searchCategory: 'NODE_VULNERABILITIES',
     attributes: {
-        ID: {
-            displayName: 'ID',
-            filterChipLabel: 'Node CVE ID',
-            searchTerm: 'CVE ID',
+        Name: {
```
Comment (pedrottimark) [inline ui/apps/platform/src/Components/CompoundSearchFilter/types.ts:326]: This change in Components folder and deletions in Containers/Vulnerabilities folder makes me hope we can refactor container-specific advanced filter types during 4.6A sprint.

**Review by pedrottimark — APPROVED** [review]
Vulnerabilities are proving practicality of pattern for advanced filters.

File: ui/apps/platform/src/Components/CompoundSearchFilter/types.ts (hunk 1 above)
Comment (dvail) [reply ui/apps/platform/src/Components/CompoundSearchFilter/types.ts:326]: For your reference, I am [re-exporting these types from a file](https://github.com/stackrox/stackrox/pull/11333/files#r1631606782) underneath `Containers/Vulnerabilities` and using that as the source of imports for everything in VM. I'm hoping this leads to minimal changes in 4.6A where we can inline the values that are currently under the `Component/` level in this top level file without creating a diff at every import point.

**Comment by dvail** [comment]
ocp-4-12-ui-e2e-tests  failures are known, existing Config Mgmt flakes

**Pull request merged**